topostats/grainstats.py

```diff
@@ -834,8 +834,8 @@ def calculate_aspect_ratio(
                     (extremes["x_max"] - extremes["x_min"]),
                     (extremes["y_max"] - extremes["y_min"]),
                 )
-                # Enforce >= 1 aspect ratio
-                if aspect_ratio < 1.0:
+                # Enforce <= 1 aspect ratio
```

Comment:
I think we don't need this, instead we can remove line `828` where `aspect_ratio` is calculated and instead calculated `aspect_ratio` as...

```suggestion
                aspect_ratio = smallest_bounding_width / smallest_bounding_length
```